--- a/src/ap/hw_features.c
+++ b/src/ap/hw_features.c
@@ -835,16 +835,9 @@
 		return -1;
 	}
 
 	if (iface->conf->channel == 14) {
-		wpa_printf(MSG_INFO, "Disable OFDM/HT/VHT on channel 14");
-		iface->conf->hw_mode = HOSTAPD_MODE_IEEE80211B;
-		iface->conf->ieee80211n = 0;
-		iface->conf->ieee80211ac = 0;
-		iface->conf->ieee80211ax = 0;
-		iface->conf->ieee80211be = 0;
+		iface->conf->hw_mode = HOSTAPD_MODE_IEEE80211G;
 	}
 
 	return 1;
